src/assets/mock-data/match.mock.data.js

```diff
@@ -32,4 +32,37 @@ const mockedMatch = {
   freeTicket: false
 };
 
-module.exports = { mockedMatch };
+const mockedMatchRunning = {
+  id: 1,
+  player1: {
```

Comment:
player1: player.id ,not the whole player

Reply:
I think in the frontend it is the whole player that gets send. @Ceecker51 ?

Reply:
it can be send this way as well of course :)

Reply:
https://confluence.student.fiw.fhws.de:8443/display/PAWS1920TTRace/Matchmaker